app/services/food_api/client.rb

```diff
@@ -0,0 +1,35 @@
+require_relative 'endpoint_helpers'
+
+class FoodApi::Client
```

Comment:
This is a great pattern and I loved how you're handling errors. It will likely be best to use this pattern on solo projects and create the service when working with others.